spec/models/druid_version_zip_spec.rb

```diff
@@ -112,11 +128,31 @@
     end
   end
 
+  describe '#parts' do # zip splits
```

Comment:
@ndushay These tests seem to already be what you are asking for.  Maybe you missed them?

Reply:
I think, again, my missing the point has to do with naming:  "parts" is perfectly clear to you, but less obvious to me.  I read 

describe "#parts"
it "lists the multiple files produced"

The test, as phrased, is essentially "show that the parts method produces the correct file names" but the concept of what I was seeking is completely subsumed ... I wouldn't think to look for it in a "#parts" describe block.

I was perhaps looking for some words closer to "ensure create_zip! creates multiple files per threshold" in a test description somewhere.

There's nothing incorrect about what you did ... but I'm not grokking the code or the tests as readily as you probably hoped.  How do we meet in the middle?

Reply:
Yeah, I get what you are saying.  I can basically copy the test and put it under `create_zip!`, but that doesn't seem like a win, since I have to test it here for this method anyway.